### PR TITLE
Adds and refines printed output to users

### PR DESCRIPTION
**Pull request by DrPaulSharp**

### CODE REVIEW

**Review by StephenNneji — COMMENTED** [review]
The new print format looks good 👍. Please see other comments

File: RATapi/run.py [hunk 1]
```diff
@@ -32,4 +42,24 @@ def run(project, controls):
         for index, value in enumerate(getattr(problem_definition, parameter_field[class_list])):
             getattr(project, class_list)[index].value = value
 
+    print("Finished RAT " + horizontal_line)
     return project, results
+
+
+def get_time_string(total_time: float) -> str:
```
[the remark this answers is not in the document]
Comment (DrPaulSharp) [reply RATapi/run.py:49]: I can see that argument, I also think "time = 1482 seconds" isn't the most useful output.

[the remark this answers is not in the document]
Comment (DrPaulSharp) [reply RATapi/run.py:49]: Maybe "time = 1482 seconds (24 minutes, 42 seconds)"?

File: RATapi/run.py [hunk 2]
```diff
@@ -15,15 +17,23 @@ def run(project, controls):
         "resolution_parameters": "resolutionParams",
     }
 
+    horizontal_line = "\u2500" * 107 + "\n"
+
     problem_definition, cells, limits, priors, cpp_controls = make_input(project, controls)
 
+    print("Starting RAT " + horizontal_line)
```
[the remark this answers is not in the document]
Comment (DrPaulSharp) [reply RATapi/run.py:24]: D'oh. On it.

File: RATapi/run.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (DrPaulSharp) [reply RATapi/run.py:49]: I'll change it to a plain seconds output for now and leave the function. Maybe decide if we want to change it later.

[the remark this answers is not in the document]
Comment (StephenNneji) [reply RATapi/run.py:54]: > Maybe "time = 1482 seconds (24 minutes, 42 seconds)"?
I think it should be one or the other as it is very easy to convert between representations. We have to find out the original purpose of the timer from @arwelHughes so we know if we should stick with the status quo. Also if we are doing the minutes, I would rather we used the datetime module https://stackoverflow.com/questions/775049/how-do-i-convert-seconds-to-hours-minutes-and-seconds and avoid the extra function and test

[the remark this answers is not in the document]
Comment (DrPaulSharp) [reply RATapi/run.py:54]: Fair enough. I skipped over the datetime module as I wasn't a fan of the output, but I'm probably just being really awkward about the whole thing.

**Pull request merged**